src/main/java/org/ibp/api/rest/dataset/DatasetResource.java

```diff
@@ -348,7 +348,7 @@ private DatasetExportService getExportFileStrategy(final String fileType) {
 		final String outFileName = file.getName();
 		final HttpHeaders headers = new HttpHeaders();
 		headers
-			.add(HttpHeaders.CONTENT_DISPOSITION, String.format("attachment; filename='%s'", FileUtils.sanitizeFileName(outFileName)));
```

Comment:
Why remove the quote from content disposition filename? I think we add the quote to escape the space from filename.

Reply:
Hi @abatac , 
Upon adding the single quote, the download file name includes includes single quote in file extension